diff --git a/ports/esp32/esp32_common.cmake b/ports/esp32/esp32_common.cmake
index e3b1b81..932c88a 100644
--- a/ports/esp32/esp32_common.cmake
+++ b/ports/esp32/esp32_common.cmake
@@ -171,6 +171,8 @@ list(APPEND IDF_COMPONENTS
     esp_driver_touch_sens
     esp_eth
     esp_event
+    esp_http_server  # Required for pyDirect httpserver
+    esp_https_server  # Required for pyDirect httpserver (HTTPS support)
     esp_hw_support
     esp_netif
     esp_partition
